assets/css/challenge2.css

```diff
@@ -1,7 +1,7 @@
 body {
   position: relative;
   background: url('../img/sky.jpg');
```

Comment:
I only see that you added jupiter.gif to img, did you forget to add the img/sky.jpg to the commit?

Reply:
@DavidMatthewFraser 
Thank you!
I only uploaded the jupiter.gif image, don't touch anything from the body style.

Reply:
Sure, I'll merge these changes and you can update that when you get the chance.
Thanks Emma!